packages/aws-cdk/lib/api/util/cloudformation/stack-status.ts

```diff
@@ -37,8 +33,8 @@ export class StackStatus {
     return this.name === 'NOT_FOUND';
   }
 
-  get isSuccess(): boolean {
-    return !this.isNotFound && !this.isRollback && !this.isFailure;
+  get isComplete(): boolean {
+    return this.name.endsWith('_COMPLETE');
   }
```

Comment:
users don't have programmatic access to the CLI yet, so these methods are only used internally. However, I think you have a good point about whether it's deceiving what `isComplete` means and maybe it's better to leave it off for now since we're not using it.

might be handy to have a method indicate whether a stack is updateable. What do you think?

Reply:
If we dont use it might as well just remove it I think.. might be confusing when we look back and get confused as to why we have it in the code anyways..

`isComplete` doesn't indicate if a stack is updateable though. If it is `Rollback_Complete` then it can only be deleted right?